packages/usa-select/src/styles/_usa-select.scss

```diff
@@ -30,6 +30,12 @@
     background-image: none;
   }
 
+  &:disabled,
+  &[aria-disabled="true"] {
+    @include u-disabled;
+    opacity: 1;
```

Comment:
I was confused when I originally saw the opacity being set. But then I checked in Chrome devtools.

![image](https://user-images.githubusercontent.com/3385219/208475798-8f93d8fe-d330-4d25-81f8-1926be03ebfe.png)


Can we add a note that this undoes a default browser style?

Reply:
Added comment in 6758d68ceb5a64b900155cde8ab9cbd4c29f910a.